src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -90,3 +92,25 @@ Update "Columns" to add `links` so you can see seen any external links related t
 
 - Columns: `issue, assignee, events, title, links`
 - Sort by: `Priority`
+
+
+### Release Health
```

Comment:
We've done sentence case on all the other ones so let's keep it consistent

```suggestion
### Release health
```

Reply:
Addressed in this commit https://github.com/getsentry/sentry-docs/pull/5083/commits/9f2e2ce97d8c0235582dbc665bc16691d5c26d36#diff-124b2f7356d01f77a1ac84d799606e8247298ade1ab46fdae30038d4afbef784R104